src/css/application.css

```diff
@@ -131,73 +134,107 @@ body {
   background-color: var(--color-red);
 }
 
-.top-bar__logo-container > svg {
-  height: 3em;
+.top-bar__logo-container {
+  margin: 0 0.4em 0 0.2em;
+  min-width: 1.6em;
+  transform: scale(1.8);
```

Comment:
Bringing in `scale` seems like a weird move here, particularly given it’s an SVG and we can size it however we want!

Seems like the main hurdle here is that we want Pop to overflow the top/bottom padding that the rest of the menu items are subject to. A couple of alternative ideas:

1. Make Pop the size we want it using the `width` property, then position it relative and add a negative `top` to offset the padding
2. Instead of giving padding to the overall top bar, just give it an explicit `height`; the use of `align-items: center` will still keep everything looking nice. And we can again just give Pop the size we want

I’m strongly partial to Option 2, because it solves another problem, which I will mention below!

Reply:
Option 2 is wayyy better than what I was doing. Done.